### PR TITLE
simplify content export diff

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
@ehelms @jturel enjoy :)

**Review by ekohl — APPROVED** [review]

**Review by ehelms — APPROVED** [review]
I did not enjoy. I find these hard to read when we jam a bunch of concerns and pipes on the same line. But if I am lucky, I won't have to read this code :)

File: bats/fb-katello-content.bats [hunk 1]
```diff
@@ -163,13 +163,10 @@ setup() {
...
-    --version="${export_version}" --fields="Repositories/Name")
-
-  import_repos=$(echo $import_repos | tr "," "\n" | sort > import_repos)
-  export_repos=$(echo $export_repos | tr "," "\n" | sort > export_repos)
+  hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}" \
+    --version="${export_version}" --fields="Repositories/Name" | tr "," "\n" | sort > export_repos
+  hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${IMPORT_ORG}" \
+    --version="${export_version}" --fields="Repositories/Name" | tr "," "\n" | sort > import_repos
 
   diff import_repos export_repos
```
Comment (ekohl) [inline bats/fb-katello-content.bats:171]: If you don't want to combine that you can move to this:
```suggestion
  diff <(cat import_repos | tr ",", "\n" | sort) <(cat export_repos | tr ",", "\n" | sort)
```

However, I think that's harder to read than this.

File: bats/fb-katello-content.bats [hunk 2]
```diff
@@ -163,13 +163,10 @@ setup() {
   tSkipIfOlderThan41
   export_version=$(hammer --output csv --no-headers content-view version list --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}"\
                --per-page=1 --fields="Version"  --order="version DESC")
-  export_repos=$(hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}" \
-    --version="${export_version}" --fields="Repositories/Name")
-  import_repos=$(hammer --output csv --no-headers content-view version show --content-view="${CONTENT_VIEW}" --organization="${IMPORT_ORG}" \
-    --version="${export_version}" --fields="Repositories/Name")
-
-  import_repos=$(echo $import_repos | tr "," "\n" | sort > import_repos)
-  export_repos=$(echo $export_repos | tr "," "\n" | sort > export_repos)
```
Comment (ekohl) [inline bats/fb-katello-content.bats:172]: This was always wrong to begin with IMHO. Assigning the variable with the same name as the filename is confusing.

**Pull request merged**